src/neptune/new/utils.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2022, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+__all__ = ["stringify_unsupported"]
+
+from typing import (
+    List,
+    Mapping,
+    Union,
+)
+
+from neptune.new.internal.utils.stringify_value import StringifyValue
+
+
+def stringify_unsupported(value) -> Union[StringifyValue, Mapping, List]:
```

Comment:
Docstring proposal:

```
"""Helper function that converts unsupported values in a collection or dictionary to strings.

Example:
    run = neptune.init_run()
    complex_dict = {"tuple": ("hi", 1), "metric": 0.87}
    run["complex_dict"] = complex_dict
    # Error: The tuple is not a supported type
    from neptune.utils import stringify_unsupported
    run["complex_dict"] = stringify_unsupported(complex_dict)
"""
```

Reply:
I wonder about `Error: ....` as currently this will work but only shows a warning message. It will raise an error after 1.0.0 release or even a little further. 

What about args and `value` parameter?

Reply:
I'm sorry but I don't understand. What and where is the error?

And I suppose to be complete we can list `value` as an arg, but IMO the description and example make it pretty clear, no?

Reply:
These line:
```
    # Error: The tuple is not a supported type
```

Reply:
I see what you mean now. Got lost thinking it referred to some actual code. I'll add something